data/tutorials/ds_05_seq.md

```diff
@@ -244,9 +246,9 @@ It can be used to produce some Fibonacci numbers:
 - : int list = [0; 1; 1; 2; 3; 5; 8; 13; 21; 34]
 ```
 Why is it so? The key difference lies in the recursive call `fibs n (n + m)`. In
-the former definition, the application is complete because `fibs` is provided all the
-arguments it expects. In the latter definition, the application is partial because the
-`()` argument is missing. Since evaluation is
+the former definition, the application is complete because `fibs` is provided
+all the arguments it expects. In the latter definition, the application is
```

Comment:
i think we should say: "is provided _with_ all the arguments" for the passive form, but maybe a native english speaker could confirm.

Reply:
@christinerose Help!

Reply:
Agreed... "with all the arguments"